### PR TITLE
Start on groups example

### PR DESCRIPTION
**Pull request by neekolas**

### CODE REVIEW

**Comment by alexrisch** [comment]
[Screen_recording_20240213_173350.webm](https://github.com/xmtp/xmtp-react-native/assets/23103150/e290e89e-59ea-49a1-96c5-96394e1088a7)

Ignore my random keyboard typing 😆

**Comment by neekolas** [comment]
❤️

**Comment by alexrisch** [comment]
Screenshot_20240214_084542

**Review by nplasterer — APPROVED** [review]
I can write a test for the canMessage stuff when I do it in iOS

File: src/lib/NativeCodecs/GroupChangeCodec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  encode(): NativeMessageContent {
+    return {}
+  }
+
+  decode(nativeContent: NativeMessageContent): GroupChangeContent {
+    return nativeContent.groupChange!
+  }
+
+  fallback(): string | undefined {
+    return 'The members of this group have changed'
```
Comment (nplasterer) [inline src/lib/NativeCodecs/GroupChangeCodec.ts:28]: Do we want fallback text? I think we set this as null in the other SDKs

**Pull request merged**